nextjs-app-v14/src/app/components/button/button.tsx

```diff
@@ -12,12 +12,28 @@ export default function Button() {
     return (
         <NavigationLayout title="Button">
         <PieButton onClick={() => setCounter(counter + 1)}>Counter: {counter}</PieButton>
+
         <PieDivider />
+
         <h2>PIE Button - with Icon and text</h2>
         <PieButton>
             <IconSearch slot="icon"></IconSearch>
             Search
         </PieButton>
+
+        <PieDivider />
+
+        <h2>PIE Button - multi-line content</h2>
+        <PieButton>
+            This button has a long text that will wrap to the next line. It‘s very long
+        </PieButton>
+
+        <PieDivider />
+
+        <h2>PIE Button - truncated content</h2>
+        <PieButton isFullWidth>
+            Thisisawordthatdoesntactuallyexistsothatwecanseehowthecontentistruncated
```

Comment:
Great opportunity to use `Rindfleischetikettierungsüberwachungsaufgabenübertragungsgesetz`

Reply:
I'll need to remember this one next time I need long text 😄